collector/fixtures/e2e-64k-page-output.txt

```diff
@@ -1812,52 +1897,52 @@ node_netstat_Udp_NoPorts 120
 node_netstat_Udp_OutDatagrams 53028
 # HELP node_network_address_assign_type address_assign_type value of /sys/class/net/<iface>.
 # TYPE node_network_address_assign_type gauge
-node_network_address_assign_type{interface="eth0"} 3
+node_network_address_assign_type{device="eth0"} 3
```

Comment:
Odd, why is this happening here. I know we changed this a while ago, but maybe this fixture file is broken in master?

Reply:
It could be my mistake. I used a Debian machine to update that file with a modified `end_to_end_test.sh`.

Reply:
I found the problem, we lost end-to-end tests that were supposed to catch this. It's been fixed, and will merge https://github.com/prometheus/node_exporter/pull/1404 to fix the file.

Reply:
Please rebase your change against master to pick up the updated fixture file.

Reply:
I rebased it with updated master, and now the PR is failing with `buildkite/node-exporter`, and I cannot access the details link. Could you look into it?

Reply:
Sure, no problem. (one of the down sides to buildkite)

Looks like you have one line change that shouldn't change.
```
-node_sockstat_TCP_mem_bytes 4096
+node_sockstat_TCP_mem_bytes 65536
```

Reply:
Fixed!